### ISSUE TITLE
java.lang.SecurityException: Requires READ_PHONE_STATE

### ISSUE THREAD

**Issue by micku7zu** [micku7zu/Locate-driver#6]
```
java.lang.SecurityException: Requires READ_PHONE_STATE: Neither user 10304 nor current process has android.permission.READ_PHONE_STATE.
    at android.os.Parcel.readException(Parcel.java:1546)
    at android.os.Parcel.readException(Parcel.java:1499)
    at com.android.internal.telephony.IPhoneSubInfo$Stub$Proxy.getGroupIdLevel1(IPhoneSubInfo.java:776)
    at android.telephony.TelephonyManager.getGroupIdLevel1(TelephonyManager.java:1964)
    at android.telephony.SmsMessage.hasEmsSupport(SmsMessage.java:826)
    at com.android.internal.telephony.gsm.SmsMessage.calculateLength(SmsMessage.java:847)
    at android.telephony.SmsMessage.fragmentText(SmsMessage.java:358)
    at android.telephony.SmsManager.divideMessage(SmsManager.java:364)
    at com.micutu.locatedriver.Services.SmsSenderService.sendSMS(SmsSenderService.java:343)
    at com.micutu.locatedriver.Services.SmsSenderService.sendLocationMessage(SmsSenderService.java:236)
    at com.micutu.locatedriver.Services.SmsSenderService.onLocationUpdated(SmsSenderService.java:155)
    at io.nlopez.smartlocation.location.providers.LocationGooglePlayServicesProvider.onLocationChanged(LocationGooglePlayServicesProvider.java:205)
    at com.google.android.gms.location.internal.zzk$zzb.handleMessage(Unknown Source)
    at android.os.Handler.dispatchMessage(Handler.java:111)
    at android.os.Looper.loop(Looper.java:194)
    at android.app.ActivityThread.main(ActivityThread.java:5627)
    at java.lang.reflect.Method.invoke(Native Method)
    at java.lang.reflect.Method.invoke(Method.java:372)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:959)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:754)
```

**Comment by micku7zu** [comment]
On some phones like Lenovo P70 with Android 5.1 it requires READ_PHONE_STATE to send SMS.

**Comment by micku7zu** [comment]
Fixed 28b078f60e5cc76c06b3f495b90972aff59e6e79
